torch/_library/abstract_impl.py

```diff
@@ -204,3 +205,38 @@ def new_dynamic_size(self, *, min=0, max=None) -> torch.SymInt:
             result, min=min, max=max
         )
         return result
+
+    def create_fake_tensor(self, tensor: torch.Tensor):
```

Comment:
nit: maybe `to_fake_tensor`? Sometimes we're not "creating" a new Tensor (if a tensor is a view on another one, and we have already fakeified the other one, then this will be a view on the fakeified tensor)